src/stirling/source_connectors/perf_profiler/perf_profiler_bpf_test.cc

```diff
@@ -251,6 +259,143 @@ class PerfProfileBPFTest : public ::testing::TestWithParam<std::filesystem::path
     VLOG(1) << std::endl;
   }
 
+  Status WritePProfProtoFile() {
```

Comment:
Im a little confused about the purpose of this in the test. It seems unrelated to testing the per profiler functionality. Is it just for debugging the test?

Reply:
This is what I had in the summary of the PR:
```
Currently, we are using this to investigate flaky tests. We plan to (in a subsequent PR) refactor the pprof export code into a library function that enables pprof export from pixel scripts or from a standalone pem.
```

Since we are using this for test debug, I figured it was a nice moment to just review the basic pprof export code. We may refactor it into a library function soon.

Let's touch base in a different forum if you want to check out the results of the flaky test investigation.